src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMasses.h

```diff
@@ -74,24 +107,41 @@ namespace OpenMS
     std::vector<double> getMassShifts() const;
     
     /**
-     * @brief returns number of mass shifts
+     * @brief returns delta masses
+     */
+    std::vector<DeltaMass> getDeltaMasses() const;
+    
+    /**
+     * @brief returns number of delta masses
      */
-    unsigned getMassShiftCount() const;
+    unsigned getDeltaMassesCount() const;
    
     /**
+     * @brief returns delta mass at position i
+     */
+    DeltaMass getDeltaMassAt(int i) const;
+    
+    /**
      * @brief returns mass shift at position i
      */
     double getMassShiftAt(int i) const;
     
+    /**
+     * @brief returns label set at position i
+     */
+    LabelSet getLabelSetAt(int i) const;
+    
     private:
    
     /**
      * @brief mass shifts between peptides
      * (including zero mass shift for first peptide)
      */
-    std::vector<double> mass_shifts_;
-      
+    std::vector<DeltaMass> delta_masses_;
```

Comment:
@timosachsenberg You suggested "MultiplexDeltaMasses aggregates DeltaMasses". So the following change?
```
std::vector<*DeltaMass> delta_masses_;
```